.github/workflows/ci.yml

```diff
@@ -485,7 +485,7 @@ jobs:
             registry.cn-hongkong.aliyuncs.com/graphscope/maxgraph_standalone_manager:${{ github.sha }} || true
 
   build-gss:
-    runs-on: ubuntu-20.04
+    runs-on: self-hosted
```

Comment:
I think you should put a comment here, that the docker need to be configured doesn't need sudo priviledge on the host.  We may add new runners in the future.